### ISSUE TITLE
Grid checkbox column with keyboard selection

### ISSUE THREAD

**Issue by gyankov** [telerik/kendo-ui-core#3706]
### Enhancement

Currently users are not able to select rows via keyboard when checkbox column is enabled.  Keyboard selection should behave as in standard selection: 

[http://demos.telerik.com/kendo-ui/grid/selection](http://demos.telerik.com/kendo-ui/grid/selection)
